otterdog/eclipse-keyple.jsonnet

```diff
@@ -403,6 +403,64 @@ orgs.newOrg('eclipse-keyple') {
         orgs.newEnvironment('github-pages'),
       ],
     },
+    orgs.newRepo('keyple-less-distributed-client-kmp-lib') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      code_scanning_default_languages+: [
+        "java-kotlin"
+      ],
+      code_scanning_default_setup_enabled: true,
+      delete_branch_on_merge: false,
+      dependabot_security_updates_enabled: true,
+      description: "Eclipse Keyple™ Project: Kotlin Multiplatform distributed remote client lib for Keyple-less terminals",
+      gh_pages_build_type: "legacy",
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      homepage: "https://keyple.org/",
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
+      },
+      branch_protection_rules: [
+        orgs.newBranchProtectionRule('master') {
```

Comment:
if not specified differently, the default branch is named main, so making a branch protection rule for master does not make much sense imho.

Instead of master you could also write $.default_branch which references the actual default branch of the repo.

Reply:
to create a branch protection for the actual default branch, you could do something like that:

```
orgs.newRepo('keyple-less-distributed-client-kmp-lib') {
  local thisRepo = self,
 ...
  branch_protection_rules: [
        orgs.newBranchProtectionRule(thisRepo.default_branch) {
```

but that a bit tedious to setup for each repo.

Unfortunately, with jsonnet you can not traverse back to the parent object, you can only access the root object using `$`.